types/crew.model.ts

```diff
@@ -0,0 +1,64 @@
+import { Club } from './club.model';
+import { DisplayStatus } from './display-status.model';
+import { Event } from './event.model';
+import { Result } from './result.model';
+import { Rower } from './rower.model';
+
+export interface Crew {
+	id: string;
+	club_id: string;
+	regatta_id: string;
+	event_id: string;
+	sub_event_id: null; // Wat is een sub_event?
+	subEventCode: string;
+	knrb_num: number;
+	knrb_sequence: number; // Wat is de sequence?
+	status: number;
+	result_status: null;
+	clubName: string;
+	displayName: string; // Wat is het verschil tussen 'displayName' en 'name'?
+	name: string;
+	shortname: string; // Wat is het verschil tussen 'shortname' en 'alternative'?
+	alternative: string;
+	sequence: number;
+	combination: null;
+	boat: string; // Wat is boat en boattype?
+	boattype: string;
+	remarks: null; // Wat is het verschil tussen 'remarks' en 'public_remarks'?
```

Comment:
Remarks voor kamprechters, public remarks komen op hoesnelwasik.nl

Reply:
Als 'remarks' voor kamprechters zijn, wat is 'remarks_for_umpire' dan?

Reply:
Oja remarks zijn voor de wedstrijdleiding,
remarks_for_umpire -> kamprechters